java-security/src/main/java/com/sap/cloud/security/token/validation/validators/OAuth2TokenKeyServiceWithCache.java

```diff
@@ -29,7 +28,7 @@
  * Decorates {@link OAuth2TokenKeyService} with a cache, which gets looked up
  * before the identity service is requested via http.
  */
-public class OAuth2TokenKeyServiceWithCache implements Cacheable {
+class OAuth2TokenKeyServiceWithCache implements Cacheable {
```

Comment:
This class has been public since November. For the cache configuration in the buildpack I do not need it. Were other People using this class? Probably not. I think we can move this to package private.